### ISSUE TITLE
[Editor] Keyboard navigation doesn't work for inserting tables

### ISSUE THREAD

**Issue by yanmariomenev** [telerik/kendo-angular#4352]
**Describe the bug**

Using keyboard navigation, the user cannot define the size and create the table. The user should be able to create a table with a specific size using only keyboard navigation.

**To Reproduce**

https://www.telerik.com/kendo-angular-ui/components/editor/keyboard-navigation/

**Recording**
Example of how the navigation should work in the Editor when creating a table:
 
https://github.com/user-attachments/assets/c696c796-f946-4551-b0fb-a1bd185189e0
